pkg/safeguards/manifestresults.go

```diff
@@ -0,0 +1,108 @@
+package safeguards
+
+import (
+	"context"
+	"embed"
+	"fmt"
+
+	api "github.com/open-policy-agent/gatekeeper/v3/apis"
+	log "github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+)
+
+// Globals
+var s = runtime.NewScheme()
+
+//go:embed lib
+var embedFS embed.FS
+
+var fc FileCrawler
+
+// primes the scheme to be able to interpret beta templates
+func init() {
+	_ = clientgoscheme.AddToScheme(s)
+	_ = api.AddToScheme(s)
+
+	selectedVersion = getLatestSafeguardsVersion()
```

Comment:
do these have to be in the init? seems like they could just be a normal variable

avoiding init when possible is usually good because it can lead to weird race conditions

Reply:
can try it out

Reply:
moved to a function that builds the scheme on retrieval